### ISSUE TITLE
open review

### ISSUE THREAD

**Issue by zhangchenkai** [zotero/translators#1979]
Zotero connector cannot grab all information from the openreview website.

For example: https://openreview.net/forum?id=B1l6qiR5F7

Openreview includes many conference papers, hope zotero connector could support this

**Comment by zuphilip** [comment]
The EM translator is already working for this example. The important data can be grabbed with that, but it is recognized as article and not conference paper. Moreover, the book title is missing. Anything else?

**Comment by zhangchenkai** [comment]
Yes, I think that's all, the others are working well. Thank you.

**Comment by gncs** [comment]
I just wanted to save the following paper on OpenReview: https://openreview.net/forum?id=B1eWbxStPH and for some reason it could not extract the authors' names correctly. Is there any chance somebody could have a look at that? Thank you.

**Comment by AbeJellinek** [comment]
We could consider adding a translator for OpenReview.net in order to handle multiples in the future, but the author name issue seems to be working fine now. The lack of a proceedings title can't be addressed on our end: what the BibTeX calls `booktitle` is actually the conference name, and the actual book title is nowhere in the metadata or on the page.
